core/templates/domain/skill/skill-creation-backend-api.service.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Change year to current year.

Reply:
This is not done?

Reply:
Sorry, I forgot to do it. I will change it again